api/alter_table_compression.md

```diff
@@ -46,7 +52,9 @@ timescaledb.compress_segmentby = '<column_name> [, ...]', timescaledb.compress_c
 
 ## Sample usage
 
-Configure a hypertable that ingests device data to use compression:
+Configure a hypertable that ingests device data to use compression. Here, because we will
+often query the hypertable about a specific device or set of devices, we segment the
+compression using the `device_id` for greater performance.
```

Comment:
Maybe we can include a link to the segmentby description.

```suggestion
compression using the `device_id` for [greater performance](https://docs.timescale.com/use-timescale/latest/compression/about-compression/#segment-by-columns).
```

Reply:
That link doesn't work for me.

It redirects here, which doesn't have anything:

https://docs.timescale.com/use-timescale/latest/compression/#segment-by-columns